debug-bar-action-and-filters-addon.php

```diff
@@ -204,6 +208,7 @@ function debug_bar_action_and_filters_addon_display_filters() {
 						$table .= '<li>[<em>' . esc_html__( 'object', 'debug-bar-actions-and-filters-addon' ) . '</em>] ' . $signature . '</li>';
 					} else {
 						// Type 8 - undetermined
+						$signature = uniqid( 'undetermined_callback_' );

```

Comment:
I see default being raw callback which leads to conversion to string on comparing and that leads to errors for objects.

We handle all callbacks types (except for future). That mean that we should not have any unknown. Then I think that uniqid is correct thing for unknown because you can't put any good rule on unknown uniqueness. Plus var_export( $single_function, true ) would show if/when developer missed something.